### PR TITLE
HPCC-22431 Update WhiteList documentation

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia <jamesdefabia@lexisnexis.com>
@jakesmith Please review (Link to PDF below)

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:

http://10.240.32.243/view/Docs%207.0/job/JimDocBuild7/CE=ubuntu-18.04-amd64-1/lastSuccessfulBuild/artifact/build-docs/Release/docs/EN_US/HPCCSystemAdministratorsGuide_EN_US-7.4.13-closedown0.pdf#page=117

### CODE REVIEW

File: docs/EN_US/HPCCSystemAdmin/SA-Mods/DaliLDAP.xml [hunk 1]
```diff
@@ -127,7 +127,10 @@
 &lt;/Environment&gt;
 </programlisting></para>
 
-    <para>Hosts can be specified by hostname or IP address.</para>
+    <para>Hosts can be specified by hostname or IP address. You can also
+    specify a range of IP address. For example:</para>
+
+    <para><programlisting>&lt;Entry hosts="192.168.0.101-104" roles="Esp"/&gt;</programlisting></para>
```
Comment (jakesmith) [inline docs/EN_US/HPCCSystemAdmin/SA-Mods/DaliLDAP.xml]: In fact specifying by range was not added, so the above should be removed.

File: docs/EN_US/HPCCSystemAdmin/SA-Mods/DaliLDAP.xml [hunk 2]
```diff
@@ -185,23 +190,20 @@
...
+    for nodes within the same environment :</para>
 
     <para><programlisting>AgentExec, DaliServer, EclAgent, EclCC, EclCCServer, EclServer, EspServer, 
 SashaServer, RoxieMaster, and ThorMaster </programlisting></para>
 
     <para>These roles correspond to components in the environment, which
-    should always be implicitly whitelisted by inclusion in the
-    environment.</para>
+    should always be implicitly whitelisted by inclusion in the environment.
+    If you need to communicate with Dali from a foreign environment, then
```
Comment (jakesmith) [inline docs/EN_US/HPCCSystemAdmin/SA-Mods/DaliLDAP.xml]: I think better worded as:
```
If nodes in foreign environments need to communicate with Dali, then  
```

File: docs/EN_US/HPCCSystemAdmin/SA-Mods/DaliLDAP.xml [hunk 3]
```diff
@@ -185,23 +190,20 @@
...
+    those nodes should be explicitly added.</para>
 
     <para>In addition, the Dali server automatically whitelists these
     roles:</para>
```
Comment (jakesmith) [inline docs/EN_US/HPCCSystemAdmin/SA-Mods/DaliLDAP.xml]: Perhaps better worded as:
```
In addition the following admin tool roles are automatically whitelisted to run on the Dali server node.
```

**Comment by jakesmith** [comment]
@JamesDeFabia - a couple of comments.

**Comment by JamesDeFabia** [comment]
@jakesmith I made the edits you requested, back to you.

**Comment by jakesmith** [comment]
@JamesDeFabia - looks good.

@ghalliday - please merge.

**Pull request merged**